gravitee-rest-api-service/src/test/java/io/gravitee/rest/api/service/ApiService_UpdateTest.java

```diff
@@ -701,7 +702,7 @@ public void shouldNotChangeLifecycleStateFromCreatedInReview() throws TechnicalE
     @Test(expected = AllowOriginNotAllowedException.class)
     public void shouldHaveAllowOriginNotAllowed() throws TechnicalException {
         prepareUpdate();
-        existingApi.getProxy().getCors().getAccessControlAllowOrigin().add("/test^");
+        existingApi.getProxy().getCors().getAccessControlAllowOrigin().add("(/test^");
```

Comment:
What is the reason for this change ?

Reply:
to test an invalid regex.
It's not possible to test that a regex created by a customer is a valid origin. We can just test if the origin is a valid address or if it's a valid regex (even if the regex represents an invalid origin).